promql/promqltest/testdata/aggregators.test

```diff
@@ -235,21 +235,25 @@ eval instant at 0m max(http_requests)
   {} 4
 
 # The histogram is ignored here so the result doesn't change but it has an info annotation now.
-eval_info instant at 0m max({job="api-server"})
+eval instant at 0m max({job="api-server"})
+  expect info
   {} 4
 
 # The histogram is ignored here so there is no result but it has an info annotation now.
-eval_info instant at 0m max(http_requests_histogram)
+eval instant at 0m max(http_requests_histogram)
+  expect info
 
 eval instant at 0m min(http_requests)
```

Comment:
Since we are explicitly interested in this _not_ creating an annotation, an `expect no_info` would make sense here.